tests/integrationv2/test_pq_handshake.py

```diff
@@ -33,109 +27,9 @@
 
 EXPECTED_RESULTS = {
     # The tuple keys have the form (client_{cipher, kem_group}, server_{cipher, kem_group})
-    (Ciphers.KMS_PQ_TLS_1_0_2019_06, Ciphers.KMS_PQ_TLS_1_0_2019_06):
-        {"cipher": "ECDHE-RSA-AES256-GCM-SHA384",
-            "kem": None, "kem_group": None},
```

Comment:
Is this comment accurate anymore, and should you just get rid of the "kem" field?

Reply:
Updated comment to be more accurate, and removed the "kem" field.